src/rospkg/environment.py

```diff
@@ -109,7 +109,7 @@ def get_ros_package_path(env=None):
     """
     if env is None:
         env = os.environ
-    return env.get(ROS_PACKAGE_PATH, None)
+    return env.get(ROS_PACKAGE_PATH, '') + ':/usr/share'
```

Comment:
I don't like the idea that we need to set an additional environment variable for this reason.

If we would do so none of the current ROS users has that environment variable set. We can push releases which update the setup files under `/opt/ros/distro` to always set that variable. But what do we do about distros which we are not releasing binary packages for anymore? Usually we try hard to maintain the ability to continue building them from source. This would imply changing behavior on those (hopefully it doesn't break anything - but I would try to avoid that).

The `RPP_NO_DEFAULTS` only addresses the case of a single environment variable. But ROS has many more and any package can contribute more using environment hooks. How are those being set on Debian? I personally think sourcing a file on Debian is the better approach.